Content/RecipeChanges/FoxRecipeChanges/Calamity.cs

```diff
@@ -0,0 +1,203 @@
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+using CalamityMod.Items.Weapons.Melee;
+using CalamityMod.Items.Weapons.Rogue;
+using CalamityMod.Items.Materials;
+using ThoriumMod.Items.HealerItems;
+using ThoriumMod.Items.BardItems;
+using ThoriumMod.Items.ThrownItems;
+using SOTS.Items.Permafrost;
+
+namespace TheBereftSouls.Content.RecipeChanges;
+
+[ExtendsFromMod("CalamityMod", "SOTS", "ThoriumMod")]
+
+public class AngelTreads : ModSystem

```

Comment:
why are these all seperate ModSystems you should put all recipe changes in the same class
for example:
```suggestion
public class RecipeUpdater: ModSystem
{
	public override void PostAddRecipes()
	{	
		foreach (var recipe in Main.recipe)
		{
			if (recipe.createItem.type != ModContent.ItemType<CalamityMod.Items.Accessories.AngelTreads>())
				recipe.AddIngredient(ModContent.ItemType<SoulOfPlight>());
			if (recipe.createItem.type != ModContent.ItemType<CalamityMod.Items.Accessories.HallowedRune>())
				recipe.AddIngredient(ModContent.ItemType<SoulOfPlight>(), 5);	
		}
	}
}
```

Reply:
this way you don't have to iterate through multiple for loops and you shouldn't need multiple ModSystem classes

Reply:
you're doing quite well without any knowledge. good job.